### DISCUSSION TITLE
Collections Order

### DISCUSSION THREAD

**Discussion by cogdog** [pixelfed/pixelfed#4779] [Ideas]
Is there, will there be options to set the order that photos appear in a collection (Newest first, oldest first, random, manual?). It looks like to be that it's oldest first now, which seems backwards from what I might expect.

My dog photos, the newest ones added appear at the end 
https://pixelfed.social/c/622934687937366947

**Comment by silsha** [comment]
There seems to be an issue for this: https://github.com/pixelfed/pixelfed/issues/2575

**Reply by cogdog** [reply]
Thanks! Added a comment there.
